src/clients/rpc_tls_client.h

```diff
@@ -60,7 +60,8 @@ class HttpRpcTlsClient : public TlsClient, public http::ResponseProcessor
     const std::string& method,
     const CBuffer params,
     const std::string& content_type,
-    llhttp_method verb)
+    llhttp_method verb,
+    const char* jwt = nullptr)
```

Comment:
>jwt [](start = 16, length = 3)

I wonder if you should name this as auth_token or id_token?

Reply:
jwt is an implementation detail. 

---
In reply to: [561288919](https://github.com/microsoft/CCF/pull/2096#discussion_r561288919) [](ancestors = 561288919)

Reply:
changed to auth_token